src/test/java/com/varchenko/BubbleSortTest.java

```diff
@@ -0,0 +1,30 @@
+package com.varchenko;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+class BubbleSortTest {
+    private final BubbleSort bubbleSort = new BubbleSort();
+
+    @Test
+    void sortArray_positiveNumbers() {
+        int[] arrayToSort = {5, 2, 1, 10, 50, 10};
+        int[] expectedResult = {1, 2, 5, 10, 10, 50};
+
+        int[] result = bubbleSort.sortArray(arrayToSort);
+System.out.println(Arrays.toString(arrayToSort));
```

Comment:
Please remove this `System.out.println`.

Also, this line has been moved to the left. In IntelliJ, you can fix it via `Ctrl + Alt + O`.

Reply:
done